plugins/CoreHome/javascripts/popover.js

```diff
@@ -142,7 +142,17 @@ var Piwik_Popover = (function () {
 
         /** Set the title of the popover */
         setTitle: function (titleHtml) {
+            var titleText = $('<div>' + titleHtml + '</div>').text();
+            if (titleText.length > 60) {
+                titleHtml = $('<span>').attr('class', 'tooltip').attr('title', titleText).html(titleHtml);
```

Comment:
Could point. I'm actually not sure what the behaviour should be but as you suggested sounds good. So it would be actually expected to remove HTML and only keep the text. Can you maybe just reuse `piwikHelper.htmlDecode()` in line 145 or is there a difference?